tests/e2e/runners/firefox/firefox.e2e.runner.js

```diff
@@ -27,6 +28,8 @@ async function loadFirefoxDriver () {
 // so we need to use the second one first
 const driver_for_gutenberg = await loadFirefoxDriver();
 const driver_for_ray_charles = await loadFirefoxDriver();
+const driver_for_tonedear = await loadFirefoxDriver();
```

Comment:
As the comment says above this block, the drivers need to be listed in reverse test order here, so that when the user runs the tests locally we see the Ray Charles first (which will be on top of the other windows). So please put the tonedear test above gutenberg in this list, and update the comment to reflect that there are three tests

Reply:
Done Sir